src/style.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable string-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url('fonts/Avenir-Heavy.woff2') format('woff2'),
+    url('fonts/Avenir-Heavy.woff') format('woff');
+  font-weight: 900;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url('fonts/Avenir-Book.woff2') format('woff2'),
+    url('fonts/Avenir-Book.woff') format('woff');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+body,
+body * {
+  font-family: 'Avenir', sans-serif;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
do not set the font-family, margin and padding to all elements on the page. choose only that you need

Reply:
Done. Fix it at new commit